packages/terra-functional-testing/tests/jest/terra-cli/wdio/test-runner.test.js

```diff
@@ -118,5 +117,25 @@ describe('Test Runner', () => {
         config: '/path', theme: 'terra-mock-theme', locale: 'fr', formFactor: 'large',
       });
     });
+
+    it('should initiate a test runner with custom hostname and port options', async () => {
+      jest.spyOn(TestRunner, 'run').mockImplementation(() => Promise.resolve());
+
+      await TestRunner.start({
+        config: '/path',
+        locales: ['en'],
+        themes: ['terra-default-theme'],
+        hostname: 'hostname',
+        port: 8080,
```

Comment:
Any reason why `baseUrl`, `suite` and `spec` are not tested here or in a separate test?

Reply:
I just chose a few options to verify they were being passed through, would you like to see a test added that adds them all?

Reply:
Not really. There are several options added but saw only a couple are included here.